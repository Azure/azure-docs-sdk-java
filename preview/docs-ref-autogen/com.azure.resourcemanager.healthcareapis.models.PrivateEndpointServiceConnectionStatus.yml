### YamlMime:JavaType
uid: "com.azure.resourcemanager.healthcareapis.models.PrivateEndpointServiceConnectionStatus"
fullName: "com.azure.resourcemanager.healthcareapis.models.PrivateEndpointServiceConnectionStatus"
name: "PrivateEndpointServiceConnectionStatus"
nameWithType: "PrivateEndpointServiceConnectionStatus"
summary: "Defines values for PrivateEndpointServiceConnectionStatus."
inheritances:
- "<xref href=\"java.lang.Object\" data-throw-if-not-resolved=\"False\" />"
- "<xref href=\"com.azure.core.util.ExpandableStringEnum\" data-throw-if-not-resolved=\"False\" />"
inheritedMembers:
- "com.azure.core.util.ExpandableStringEnum.<T>fromString(java.lang.String,java.lang.Class<T>)"
- "com.azure.core.util.ExpandableStringEnum.<T>values(java.lang.Class<T>)"
- "com.azure.core.util.ExpandableStringEnum.equals(java.lang.Object)"
- "com.azure.core.util.ExpandableStringEnum.hashCode()"
- "com.azure.core.util.ExpandableStringEnum.toString()"
- "java.lang.Object.clone()"
- "java.lang.Object.finalize()"
- "java.lang.Object.getClass()"
- "java.lang.Object.notify()"
- "java.lang.Object.notifyAll()"
- "java.lang.Object.wait()"
- "java.lang.Object.wait(long)"
- "java.lang.Object.wait(long,int)"
syntax: "public final class PrivateEndpointServiceConnectionStatus extends ExpandableStringEnum<PrivateEndpointServiceConnectionStatus>"
constructors:
- "com.azure.resourcemanager.healthcareapis.models.PrivateEndpointServiceConnectionStatus.PrivateEndpointServiceConnectionStatus()"
fields:
- "com.azure.resourcemanager.healthcareapis.models.PrivateEndpointServiceConnectionStatus.APPROVED"
- "com.azure.resourcemanager.healthcareapis.models.PrivateEndpointServiceConnectionStatus.PENDING"
- "com.azure.resourcemanager.healthcareapis.models.PrivateEndpointServiceConnectionStatus.REJECTED"
methods:
- "com.azure.resourcemanager.healthcareapis.models.PrivateEndpointServiceConnectionStatus.fromString(java.lang.String)"
- "com.azure.resourcemanager.healthcareapis.models.PrivateEndpointServiceConnectionStatus.values()"
type: "class"
metadata: {}
package: "com.azure.resourcemanager.healthcareapis.models"
artifact: com.azure.resourcemanager:azure-resourcemanager-healthcareapis:1.0.0-beta.1
