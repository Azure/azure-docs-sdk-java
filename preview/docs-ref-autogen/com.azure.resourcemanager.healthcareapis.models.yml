### YamlMime:JavaPackage
uid: "com.azure.resourcemanager.healthcareapis.models"
fullName: "com.azure.resourcemanager.healthcareapis.models"
name: "com.azure.resourcemanager.healthcareapis.models"
summary: "Package containing the data models for HealthcareApisManagementClient. Azure Healthcare APIs Client."
classes:
- "com.azure.resourcemanager.healthcareapis.models.CheckNameAvailabilityParameters"
- "com.azure.resourcemanager.healthcareapis.models.ManagedServiceIdentityType"
- "com.azure.resourcemanager.healthcareapis.models.OperationDisplay"
- "com.azure.resourcemanager.healthcareapis.models.OperationListResult"
- "com.azure.resourcemanager.healthcareapis.models.OperationResultStatus"
- "com.azure.resourcemanager.healthcareapis.models.PrivateEndpoint"
- "com.azure.resourcemanager.healthcareapis.models.PrivateEndpointConnection"
- "com.azure.resourcemanager.healthcareapis.models.PrivateEndpointConnectionListResultDescription"
- "com.azure.resourcemanager.healthcareapis.models.PrivateEndpointConnectionProvisioningState"
- "com.azure.resourcemanager.healthcareapis.models.PrivateEndpointServiceConnectionStatus"
- "com.azure.resourcemanager.healthcareapis.models.PrivateLinkResource"
- "com.azure.resourcemanager.healthcareapis.models.PrivateLinkServiceConnectionState"
- "com.azure.resourcemanager.healthcareapis.models.ProvisioningState"
- "com.azure.resourcemanager.healthcareapis.models.PublicNetworkAccess"
- "com.azure.resourcemanager.healthcareapis.models.ServiceAccessPolicyEntry"
- "com.azure.resourcemanager.healthcareapis.models.ServiceAcrConfigurationInfo"
- "com.azure.resourcemanager.healthcareapis.models.ServiceAuthenticationConfigurationInfo"
- "com.azure.resourcemanager.healthcareapis.models.ServiceCorsConfigurationInfo"
- "com.azure.resourcemanager.healthcareapis.models.ServiceCosmosDbConfigurationInfo"
- "com.azure.resourcemanager.healthcareapis.models.ServiceExportConfigurationInfo"
- "com.azure.resourcemanager.healthcareapis.models.ServicesDescriptionListResult"
- "com.azure.resourcemanager.healthcareapis.models.ServicesPatchDescription"
- "com.azure.resourcemanager.healthcareapis.models.ServicesProperties"
- "com.azure.resourcemanager.healthcareapis.models.ServicesResource"
- "com.azure.resourcemanager.healthcareapis.models.ServicesResourceIdentity"
enums:
- "com.azure.resourcemanager.healthcareapis.models.Kind"
- "com.azure.resourcemanager.healthcareapis.models.ServiceNameUnavailabilityReason"
interfaces:
- "com.azure.resourcemanager.healthcareapis.models.Operation"
- "com.azure.resourcemanager.healthcareapis.models.OperationResults"
- "com.azure.resourcemanager.healthcareapis.models.OperationResultsDescription"
- "com.azure.resourcemanager.healthcareapis.models.Operations"
- "com.azure.resourcemanager.healthcareapis.models.PrivateEndpointConnectionDescription"
- "com.azure.resourcemanager.healthcareapis.models.PrivateEndpointConnectionDescription.Definition"
- "com.azure.resourcemanager.healthcareapis.models.PrivateEndpointConnectionDescription.DefinitionStages"
- "com.azure.resourcemanager.healthcareapis.models.PrivateEndpointConnectionDescription.DefinitionStages.Blank"
- "com.azure.resourcemanager.healthcareapis.models.PrivateEndpointConnectionDescription.DefinitionStages.WithCreate"
- "com.azure.resourcemanager.healthcareapis.models.PrivateEndpointConnectionDescription.DefinitionStages.WithParentResource"
- "com.azure.resourcemanager.healthcareapis.models.PrivateEndpointConnectionDescription.DefinitionStages.WithPrivateEndpoint"
- "com.azure.resourcemanager.healthcareapis.models.PrivateEndpointConnectionDescription.DefinitionStages.WithPrivateLinkServiceConnectionState"
- "com.azure.resourcemanager.healthcareapis.models.PrivateEndpointConnectionDescription.Update"
- "com.azure.resourcemanager.healthcareapis.models.PrivateEndpointConnectionDescription.UpdateStages"
- "com.azure.resourcemanager.healthcareapis.models.PrivateEndpointConnectionDescription.UpdateStages.WithPrivateEndpoint"
- "com.azure.resourcemanager.healthcareapis.models.PrivateEndpointConnectionDescription.UpdateStages.WithPrivateLinkServiceConnectionState"
- "com.azure.resourcemanager.healthcareapis.models.PrivateEndpointConnections"
- "com.azure.resourcemanager.healthcareapis.models.PrivateLinkResourceDescription"
- "com.azure.resourcemanager.healthcareapis.models.PrivateLinkResourceListResultDescription"
- "com.azure.resourcemanager.healthcareapis.models.PrivateLinkResources"
- "com.azure.resourcemanager.healthcareapis.models.Services"
- "com.azure.resourcemanager.healthcareapis.models.ServicesDescription"
- "com.azure.resourcemanager.healthcareapis.models.ServicesDescription.Definition"
- "com.azure.resourcemanager.healthcareapis.models.ServicesDescription.DefinitionStages"
- "com.azure.resourcemanager.healthcareapis.models.ServicesDescription.DefinitionStages.Blank"
- "com.azure.resourcemanager.healthcareapis.models.ServicesDescription.DefinitionStages.WithCreate"
- "com.azure.resourcemanager.healthcareapis.models.ServicesDescription.DefinitionStages.WithEtag"
- "com.azure.resourcemanager.healthcareapis.models.ServicesDescription.DefinitionStages.WithIdentity"
- "com.azure.resourcemanager.healthcareapis.models.ServicesDescription.DefinitionStages.WithKind"
- "com.azure.resourcemanager.healthcareapis.models.ServicesDescription.DefinitionStages.WithLocation"
- "com.azure.resourcemanager.healthcareapis.models.ServicesDescription.DefinitionStages.WithProperties"
- "com.azure.resourcemanager.healthcareapis.models.ServicesDescription.DefinitionStages.WithResourceGroup"
- "com.azure.resourcemanager.healthcareapis.models.ServicesDescription.DefinitionStages.WithTags"
- "com.azure.resourcemanager.healthcareapis.models.ServicesDescription.Update"
- "com.azure.resourcemanager.healthcareapis.models.ServicesDescription.UpdateStages"
- "com.azure.resourcemanager.healthcareapis.models.ServicesDescription.UpdateStages.WithPublicNetworkAccess"
- "com.azure.resourcemanager.healthcareapis.models.ServicesDescription.UpdateStages.WithTags"
- "com.azure.resourcemanager.healthcareapis.models.ServicesNameAvailabilityInfo"
metadata: {}
package: "com.azure.resourcemanager.healthcareapis.models"
artifact: com.azure.resourcemanager:azure-resourcemanager-healthcareapis:1.0.0-beta.1
