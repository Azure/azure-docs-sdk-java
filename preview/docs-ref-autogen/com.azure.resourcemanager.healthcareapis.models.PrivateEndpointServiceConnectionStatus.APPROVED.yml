### YamlMime:JavaMember
uid: "com.azure.resourcemanager.healthcareapis.models.PrivateEndpointServiceConnectionStatus.APPROVED*"
fullName: "com.azure.resourcemanager.healthcareapis.models.PrivateEndpointServiceConnectionStatus.APPROVED"
name: "APPROVED"
nameWithType: "PrivateEndpointServiceConnectionStatus.APPROVED"
members:
- uid: "com.azure.resourcemanager.healthcareapis.models.PrivateEndpointServiceConnectionStatus.APPROVED"
  fullName: "com.azure.resourcemanager.healthcareapis.models.PrivateEndpointServiceConnectionStatus.APPROVED"
  name: "APPROVED"
  nameWithType: "PrivateEndpointServiceConnectionStatus.APPROVED"
  summary: "Static value Approved for PrivateEndpointServiceConnectionStatus."
  syntax: "public static final PrivateEndpointServiceConnectionStatus APPROVED"
type: "field"
metadata: {}
package: "com.azure.resourcemanager.healthcareapis.models"
artifact: com.azure.resourcemanager:azure-resourcemanager-healthcareapis:1.0.0-beta.1
